src/main/java/org/rumbledb/compiler/TranslationVisitor.java

```diff
@@ -248,39 +245,39 @@ else if (content instanceof JsoniqParser.TypeSwitchExprContext)
     // TODO [EXPRVISITOR] count
     @Override
     public Void visitFlowrExpr(JsoniqParser.FlowrExprContext ctx) {
-        FlworClause startClause, childClause;
-        List<FlworClause> contentClauses = new ArrayList<>();
+        Clause startClause, childClause;
```

Comment:
Google's java style guide is strictly in favor of using [one variable per declaration](https://google.github.io/styleguide/javaguide.html#s4.8.2-variable-declarations). Could we put these variables on different lines?

Reply:
Agreed. DONE.